bokehjs/src/less/widgets/font-awesome.less

```diff
@@ -1,16 +1,16 @@
-@font-awesome: "../../vendor/font-awesome-4.2.0/less";
+@font-awesome: "../../../node_modules/font-awesome/less";
 
 @import "@{font-awesome}/variables.less";
 @fa-css-prefix: bk-fa;
-@fa-font-path:  "/bokehjs/static/js/vendor/font-awesome-4.2.0/fonts";
```

Comment:
I'm not sure why this was here, because we inline fonts in `./fonts.less` and don't use any paths to fonts.